tracer/test/Datadog.Trace.Tests/DataStreamsMonitoring/HashHelperTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// <copyright file="HashHelperTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Datadog.Trace.DataStreamsMonitoring.Hashes;
+using Datadog.Trace.Util;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.Tests.DataStreamsMonitoring;
+
+public class HashHelperTests
```

Comment:
Not necessary for this PR obv, more to start the discussion, those tests could be good candidates for property based testing, wdyt?

Reply:
Yeah, could do